codec-http/src/main/java/io/netty/handler/codec/http/DefaultHttpHeaders.java

```diff
@@ -324,7 +324,7 @@ public int size() {
 
     @Override
     public boolean contains(String name, String value, boolean ignoreCase) {
-        return contains((CharSequence) name, (CharSequence) value, ignoreCase);
+        return contains(name, value, ignoreCase);
```

Comment:
This is needed as otherwise it will go into an infinite loop.

Reply:
yes, it is my fault.